Paintroid/src/main/res/layout/dialog_pocketpaint_shapes.xml

```diff
@@ -17,23 +17,133 @@
 
+    <TextView
+        android:id="@+id/pocketpaint_outline_view_text_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="4dp"
+        android:text="@string/dialog_brush_width_text"
+        android:textAllCaps="true"
+        android:textColor="@color/pocketpaint_main_rectangle_tool_accent_color"
+        android:textStyle="bold"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="24dip" />
+
+    <View
+        android:id="@+id/pocketpaint_outline_view_border"
+        android:layout_width="match_parent"
+        android:layout_height="2dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:background="@color/pocketpaint_main_separator_color"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="16dp" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:weightSum="5"
+        android:orientation="horizontal" >
+
+        <SeekBar
+            android:id="@+id/pocketpaint_shape_stroke_width_seek_bar"
+            android:progressTint="@color/pocketpaint_main_rectangle_tool_accent_color"
+            android:thumbTint="@color/pocketpaint_main_rectangle_tool_accent_color"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="4"
+            android:layout_gravity="center"
+            android:layout_marginBottom="5dip"
+            android:layout_marginTop="16dp"
+            android:max="100"
+            android:minHeight="30dip"
+            android:layout_marginEnd="16dp"
+            android:layout_marginStart="24dp" />
+
+        <EditText
+            android:id="@+id/pocketpaint_shape_outline_edit"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="1"
+            android:layout_gravity="center"
+            android:layout_marginTop="16dp"
+            android:inputType="number"
+            android:imeOptions="actionDone"
+            android:textColor="@color/pocketpaint_main_rectangle_tool_accent_color"
+            android:textSize="14sp"
+            android:textStyle="bold"
+            tools:ignore="LabelFor" />
+    </LinearLayout>
+
+    <TextView
+        android:id="@+id/pocketpaint_shape_tool_fill_outline"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="4dp"
+        android:layout_marginTop="16dp"
+        android:text="@string/shape_tool_dialog_fill_title"
+        android:textAllCaps="true"
+        android:textColor="@color/pocketpaint_main_rectangle_tool_accent_color"
+        android:textStyle="bold"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="24dip" />
+
+    <View
```

Comment:
Please use the `<View style="@style/PocketPaintDividerHorizontal"/>` to avoid duplication.